### PR TITLE
Switch back to the original plone.formwidget.autocomplete package.

### LINKED ISSUE #1152: plone.formwidget.autocomplete durch eigenes Widget ablösen
Unser eigener Fork von `plone.formwidget.autocomplete` ist nicht mit dem Upstream nachgeführt, und verursacht einige Probleme (siehe z.B. #1121).

Wir sollten hier stattdessen unser eigenes Widget implementieren.

### PR DESCRIPTION
**Pull request by phgross**
Since the OGIP 15 task responsible rework, it's no longer necessary to dynamically change the data source on task responsible fields. Therefore our fork of the `plone.formwidget.autocomplete` is no longer necessary.

This change should allow us to switch to the latests plone 4.3.x release.

Styles: https://github.com/4teamwork/plonetheme.teamraum/pull/555

Closes #1152

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changelog
 2017.3.0 (unreleased)
 ---------------------
 
+- Switch back to the orginal plone.formwidget.autocomplete package from our fork. [phgross]
+
```
Comment (deiferni) [inline docs/HISTORY.txt]: ✂️

File: opengever/core/upgrades/20170608134918_change_to_original_plone_formwidget_autocomplete/jsregistry.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<object name="portal_javascripts" meta_type="JavaScripts Registry">
+
+  <javascript
+      id="++resource++plone.formwidget.autocomplete/jquery.autocomplete.min.js"
+      insert-after="collective.js.jqueryui.custom.min.js"
+      />
+
+  <javascript
+      id="++resource++plone.formwidget.autocomplete/formwidget-autocomplete.js"
+      insert-after="collective.js.jqueryui.custom.min.js"
```
Comment (deiferni) [inline opengever/core/upgrades/20170608134918_change_to_original_plone_formwidget_autocomplete/jsregistry.xml]: won't this insert `formwidget-autocomplete.js` before `jquery.autocomplete.min.js` 🤔  ?

File: opengever/core/profiles/default/jsregistry.xml [hunk 3]
```diff
@@ -412,4 +412,14 @@
       inline="False"
       />
 
+  <javascript
+      id="++resource++plone.formwidget.autocomplete/jquery.autocomplete.min.js"
```
Comment (deiferni) [inline opengever/core/profiles/default/jsregistry.xml]: why is this necessary? for a fresh installation the js files should already be registered correctly when the profile `plone.formwidget.autocomplete:default` is installed, right?

Comment (phgross) [inline opengever/core/profiles/default/jsregistry.xml]: its jus't ordering not a complete registration.
